### ISSUE TITLE
Support slot types other than literal (e.g. Date)

### ISSUE THREAD

**Issue by tomdotorg** [jjaquinta/EchoSim#2]
Hi, Thanks for providing this great software for us to use. It is a billion times better than talking to an echo endlessly!

And also thank you for being the first to write a book on developing for the Echo. I am a proud owner of the book too.

This capability might exist already, and if so, sorry for not being able to find it in the sources.

I have a Date slot for my application. Prior to using your software, I was doing it the manual way and the various words passed into the Echo ("today", "last Wednesday", etc.) were showing up as ISO dates. Using EchoSim, I am getting the literal words passed into my application, which does not parse well. :)

Do you plan to add this or should I see that as a limitation to be worked around, or implemented? I can see how it might be very tricky to code and keep up with Amazon's implementation of the Date Slot Type.

Thanks again!

**Comment by jjaquinta** [comment]
Good point. I'll need to do some test apps. I've never worked with a Date slot. But it is something that EchoSim should be able to do. I'll work to add it.

**Comment by tomdotorg** [comment]
Thanks, that's fantastic. Once you commit something that puts an architectural stake in the ground, I'll see if I can pitch in and help out.
